public/content/sv/welcome/links-documentation.md

```diff
@@ -0,0 +1,17 @@
+# Länkar och dokumentation
+
+Efter att ha avslutat rundtuden, kan du kolla in några av de följande länkarna för mer information:
+
+**NOTERA:** Följande länkar är på engelska
+
+* [Lära sig D som en nybörjare](http://ddili.org/ders/d.en/index.html)
+* [Lära sig D för en erfaren programmerare](http://wiki.dlang.org/Coming_From) från andra språk
```

Comment:
I agree that it sounded a bit strange.
I think your suggestions would be better.